docs/content/connectors/mysql-cdc.md

```diff
@@ -463,36 +475,119 @@ Data Type Mapping
     </tr>
     <tr>
       <td>TIME [(p)]</td>
-      <td>TIME [(p)] [WITHOUT TIMEZONE]</td>
+      <td>TIME [(p)]</td>
     </tr>
     <tr>
       <td>DATETIME [(p)]</td>
-      <td>TIMESTAMP [(p)] [WITHOUT TIMEZONE]</td>
+      <td>TIMESTAMP [(p)]</td>
     </tr>
     <tr>
       <td>TIMESTAMP [(p)]</td>
-      <td>TIMESTAMP [(p)]<br>
-          TIMESTAMP [(p)] WITH LOCAL TIME ZONE
+      <td>TIMESTAMP [(p)]
+      </td>
+    </tr>
+    <tr>
+      <td>
+        CHAR(n)
+      </td>
+      <td>CHAR(n)</td>
+    </tr>
+    <tr>
+      <td>
+        VARCHAR(n)
+      </td>
+      <td>VARCHAR(n)</td>
+    </tr>
+    <tr>
+      <td>
+        BIT(n)<br>
+        where n > 1
+      </td>
+      <td>BINARY(⌊n/8⌋)</td>
```

Comment:
`⌊n/8⌋` is a special math signal ?

Reply:
yes.
But I use the wrong math signal, we should use ⌈n/8⌉, which represents ceil(n/8)